lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1792,6 +1792,8 @@ namespace Js
             Wasm::BaseWasmReader *reader = nullptr;
             Wasm::WasmBytecodeGenerator *bytecodeGen = nullptr;
 
+            bool traceReader = PHASE_TRACE1(WasmReaderPhase);
```

Comment:
You mean to remove the trace param in the reader constructor and simply use `PHASE_TRACE1(WasmReaderPhase)` in the WasmReader ?